go.mod

```diff
@@ -4,11 +4,11 @@ require (
 	github.com/DATA-DOG/go-sqlmock v1.5.0
 	github.com/go-kit/log v0.2.0
 	github.com/go-sql-driver/mysql v1.6.0
+	github.com/gofrs/uuid v4.2.0+incompatible
```

Comment:
So it seems like this fork dropped support Go modules, which seems a bit silly.

Maybe we should use `github.com/google/uuid` instead? The only difference is we need to use `uuid.Parse()` instead of `uuid.FromString()`.

Reply:
Thanks for the suggestion, indeed it seems a safer alternative. I will do it!